docs/data/sdks/ios-swift/ampli.md

```diff
@@ -0,0 +1,205 @@
+---
+title: iOS Ampli Wrapper
+description: Learn how to install and use the Ampli Wrapper for the iOS Swift runtime. 
+---
+
+--8<-- "includes/ampli/ampli-overview-section.md"
+
+Amplitude Data supports tracking analytics events from iOS apps written in Swift.
+
+!!!warning "Objective-C Not Supported"
+    Please note that the [latest iOS SDK](/data/sdks/ios-swift) is NOT compatible with Objective-C projects. Use the [maintenance iOS SDK](/data/sdks/ios) with [maintenance Ampli](/data/sdks/ios/ampli) if your project requires compatibility with Objective-C.
+
+!!!info "Ampli iOS Resources"
+    [:material-language-swift: Swift Example](https://github.com/amplitude/ampli-examples/tree/main/ios/swift/v2/AmpliSwiftSampleApp) · [:material-code-tags-check: Releases](https://www.npmjs.com/package/@amplitude/ampli?activeTab=versions)
```

Comment:
The swift example is 404
![image](https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/31029607/59b65344-0e97-4846-9e27-75f0a1d5b7ee)

Reply:
Thanks, this will be the correct link. I plan to [merge the examples](https://github.com/amplitude/ampli-examples/pull/268) before this doc update.